modules/javafx.graphics/src/main/native-glass/gtk/glass_window.h

```diff
@@ -183,21 +121,25 @@ class WindowContextBase: public WindowContext {
         bool on_key_event;
     } im_ctx;
 
-    size_t events_processing_cnt;
-    bool can_be_deleted;
-protected:
-    std::set<WindowContextTop*> children;
+    size_t events_processing_cnt{};
+
+    std::set<WindowContext*> children;
     jobject jwindow;
-    jobject jview;
-    GtkWidget* gtk_widget;
-    GdkWindow* gdk_window = NULL;
-    GdkWMFunction gdk_windowManagerFunctions;
+    jobject jview{};
+
+    GdkWindow *gdk_window{};
+    GdkWMFunction initial_wmf;
+    GdkWMFunction current_wmf;
 
-    bool is_iconified;
-    bool is_maximized;
     bool is_mouse_entered;
     bool is_disabled;
+    bool on_top;
+    bool can_be_deleted;
+    bool mapped;
 
+    gint initial_state_mask;
+protected:
+protected:
```

Comment:
`protected:` is doubled here, you could remove one of those

Reply:
Fixed